src/main/java/io/vavr/control/Validation.java

```diff
@@ -111,6 +111,15 @@ private Validation() {
     /**
      * Creates a {@code Validation} of an {@code Either}.
      *
+     * <pre>{@code
+     * // validates the either type if either is right then validation
+     * //contains right value otherwise throws NoSuchElementException
+     * // Creates Validation instance with the value "vavr"
+     *    Validation validation = Validation.fromEither(Either.right("vavr"))
```

Comment:
Please also add the missing generics, I think `<?, String>` should compile in this case...